server/sonar-web/src/main/js/components/issue/issue-view.js

```diff
@@ -78,7 +78,7 @@ export default Marionette.ItemView.extend({
     const key = this.model.get('key');
     const componentUuid = this.model.get('componentUuid');
     this.model.reset({ key, componentUuid }, { silent: true });
-    return this.model.fetch(options).done(function () {
+    return this.model.fetch(options).done(() => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)